commit 8483f3b7f4243f8c871d3d93e53ec11561088c18
Author: Christian König <christian.koenig@amd.com>
Date:   Thu Aug 16 12:01:03 2018 +0200

    drm/amdgpu: fix VM clearing for the root PD
    
    [ Upstream commit 8604ffcbf04f8f4f3f55a9e46e5ff948b2ed4290 ]
    
    We need to figure out the address after validating the BO, not before.
    
    Signed-off-by: Christian König <christian.koenig@amd.com>
    Reviewed-by: Felix Kuehling <Felix.Kuehling@amd.com>
    Reviewed-by: Junwei Zhang <Jerry.Zhang@amd.com>
    Reviewed-by: Huang Rui <ray.huang@amd.com>
    Signed-off-by: Alex Deucher <alexander.deucher@amd.com>
    Signed-off-by: Sasha Levin <alexander.levin@microsoft.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

diff --git a/drivers/gpu/drm/amd/amdgpu/amdgpu_vm.c b/drivers/gpu/drm/amd/amdgpu/amdgpu_vm.c
index fdcb498f6d19..bbd4daa4d870 100644
--- a/drivers/gpu/drm/amd/amdgpu/amdgpu_vm.c
+++ b/drivers/gpu/drm/amd/amdgpu/amdgpu_vm.c
@@ -303,7 +303,6 @@ static int amdgpu_vm_clear_bo(struct amdgpu_device *adev,
 	uint64_t addr;
 	int r;
 
-	addr = amdgpu_bo_gpu_offset(bo);
 	entries = amdgpu_bo_size(bo) / 8;
 
 	if (pte_support_ats) {
@@ -335,6 +334,7 @@ static int amdgpu_vm_clear_bo(struct amdgpu_device *adev,
 	if (r)
 		goto error;
 
+	addr = amdgpu_bo_gpu_offset(bo);
 	if (ats_entries) {
 		uint64_t ats_value;
 
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1274_8483f3b7f4243f8c871d3d93e53ec11561088c18.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1274_8483f3b7f4243f8c871d3d93e53ec11561088c18.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
